docs/extensions/community.md

```diff
@@ -0,0 +1,9 @@
+# Community Extensions
+
+This file should have all the known Plates extensions listed.
```

Comment:
change the above to: `This is a list of all the known community extensions for the Plates.`

And then remove the newlines between this sentence and the paragraph below.

Reply:
And create just a long single line?